publish-service/src/test/resources/expectedParsedEvents/expected_parsed_EiffelArtifactCreatedEvent.json

```diff
@@ -1 +1,51 @@
-{"msgParams":{"meta":{"type":"EiffelArtifactCreatedEvent","version":"1.1.0","tags":["my_meta[0]tags","my_meta[1]tags"],"source":{"domainId":"my_meta.source.domainId","host":"my_meta.source.host","name":"my_meta.source.name","uri":"my_meta.source.uri"},"security":{"sdm":{"authorIdentity":"my_meta.security.sdm.authorIdentity","encryptedDigest":"my_meta.security.sdm.encryptedDigest"}}}},"eventParams":{"data":{"gav":{"groupId":"my_data.gav.groupId","artifactId":"my_data.gav.artifactId","version":"my_data.gav.version"},"fileInformation":[{"classifier":"my_data.fileInformation[0].classifier","extension":"my_data.fileInformation[0].extension"},{"classifier":"my_data.fileInformation[1].classifier","extension":"my_data.fileInformation[1].extension"}],"buildCommand":"my_data.buildCommand","requiresImplementation":"NONE","name":"my_data.name","dependsOn":[{"groupId":"my_data.name.dependsOn[0].groupId","artifactId":"my_data.name.dependsOn[0].artifactId","version":"my_data.name.dependsOn[0].version"},{"groupId":"my_data.name.dependsOn[1].groupId","artifactId":"my_data.name.dependsOn[1].artifactId","version":"my_data.name.dependsOn[0].version"}],"implements":[{"groupId":"my_data.name.dependsOn[0].groupId","artifactId":"my_data.name.dependsOn[0].artifactId","version":"my_data.name.dependsOn[0].version"},{"groupId":"my_data.name.dependsOn[1].groupId","artifactId":"my_data.name.dependsOn[1].artifactId","version":"my_data.name.dependsOn[1].version"}],"customData":[{"key":"my.data.customData[0]key","value":"my.data.customData[0]value"},{"key":"my.data.customData[1]key","value":"my.data.customData[1]value"}]},"links":[{"type":"COMPOSITION","target":"e269b37d-17a1-4a10-aafb-c108735ee51f"},{"type":"ENVIRONMENT","target":"e269b37d-17a1-4a10-aafb-c108735ee50a"},{"type":"PREVIOUS_VERSION","target":"e269b37d-17a1-4a10-aafb-c108735ee50a"},{"type":"CONTEXT","target":"e269b37d-17a1-4a10-aafb-c108735ee50a"},{"type":"FLOW_CONTEXT","target":"e269b37d-17a1-4a10-aafb-c108735ee48a"}]}}
+{
```

Comment:
File is invalid. when i try to format i get these errors.
Error:Invalid character found at position 34.[Code 180, Structure 63]
Error:Expecting comma or ], not string.[Code 10, Structure 64]
Error:Strings should be wrapped in double quotes.[Code 17, Structure 64]
Error:Expecting comma or ], not colon.[Code 10, Structure 65]
Error:Expecting comma or ], not colon.[Code 10, Structure 69]
Error:Expecting comma or ], not colon.[Code 10, Structure 73]
Error:Expecting comma or ], not colon.[Code 10, Structure 77]
Error:Expecting comma or ], not string.[Code 10, Structure 80]
Error:Strings should be wrapped in double quotes.[Code 17, Structure 80]
Error:Expecting comma or ], not colon.[Code 10, Structure 81]
Error:Strings should be wrapped in double quotes.[Code 17, Structure 82]
Error:Expecting comma or ], not colon.[Code 10, Structure 86]
Error:Expecting comma or ], not colon.[Code 10, Structure 92]
Error:Expecting comma or ], not }.[Code 6, Structure 114]
Error:Expecting comma or ], not }.[Code 6, Structure 170]
Error:Expecting closing } at end[Code 22, Structure 170]
Error:Expecting closing } at end[Code 22, Structure 170]
Error:Expecting closing } at end[Code 22, Structure 170]